core/geometry/src/geometry3d/Transform.ts

```diff
@@ -740,28 +762,47 @@ export class Transform implements BeJSONFunctions {
     );
   }
   /**
-   * Initialize transforms that map each direction of a box (axis aligned) to `[0,1]`.
-   * * The corner coordinates do _not_ need to be in order in any of the x,y,z directions.
-   * * The npcToGlobalTransform (if supplied) maps 000 to the point named point000.
-   * * The npcToGlobalTransform (if supplied) maps 11 to the point named point000.
-   * * The globalToNpc transform is the inverse.
-   * @param min the "000" corner of the box
-   * @param max the "111" corner of the box
-   * @param npcToGlobal (object created by caller, re-initialized here) transform that carries 01 coordinates into the min,max box.
-   * @param globalToNpc (object created by caller, re-initialized here) transform that carries world coordinates into 01
+   * Initialize 2 Transforms: First Transform maps a box (axis aligned) specified by `min` and `max` to
+   * the unit box specified by 000 and 111 and inverse of it. Second Transform is the reverse of first.
+   * @param min the min corner of the box
+   * @param max the max corner of the box
+   * @param npcToGlobal maps global (the unit box specified by 000 and 111) to NPC (a box specified by `min`
+   * and `max`). Object created by caller, re-initialized here.
+   * @param globalToNpc maps NPC (a box specified by `min` and `max`) to global (the unit box specified by
+   * 000 and 111). Object created by caller, re-initialized here.
```

Comment:
We talked about this, so you probably have fixed this in the next PR. This is one way to say it succinctly: 
```suggestion
   * Initialize 2 Transforms that map between the unit box (specified by 000 and 111) and the range box specified by the input points.
   * @param min the min corner of the range box
   * @param max the max corner of the range box
   * @param npcToGlobal maps NPC coordinates into range box coordinates. Specifically, 000 maps to `min` and 111 maps to `max`. This Transform is the inverse of globalToNpc. Object created by caller, re-initialized here.
   * @param globalToNpc maps range box coordinates into NPC coordinates. Specifically, `min` maps to 000 and `max` maps to 111. This Transform is the inverse of npcToGlobal. Object created by caller, re-initialized here.
```